src/thunderbots/shared/util.h

```diff
@@ -1,4 +1,11 @@
 #pragma once
+#include <math.h>
 
-// TODO: Implement coordinate transform function
-// https://github.com/UBC-Thunderbots/Software/issues/20
\ No newline at end of file
+typedef struct
+{
+    float x, y;
+} Vector2D;
+
+float dot2D(Vector2D first, Vector2D second);
+
+Vector2D toLocalCoords(Vector2D point, float orientationAngle);
```

Comment:
javadoc comments

Also since we'll only be using this in the context of projecting to a Robot's coordinates, can we rename the function to `toRobotLocalCoords` or something similar?

Reply:
renamed

Reply:
yes